src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/reference/Cas2ApplicationStatusSeeding.kt

```diff
@@ -18,17 +18,23 @@ object Cas2ApplicationStatusSeeding {
         label = "Awaiting decision",
         description = "The CAS-2 team has the information they need and will make a decision.",
       ),
+      Cas2PersistedApplicationStatus(
+        id = UUID.fromString("a919097d-b324-471c-9834-756f255e87ea"),
+        name = "onWaitingList",
+        label = "On waiting list",
+        description = "The applicant has been added to the waiting list for Short-Term Accommodation (CAS-2).",
+      ),
       Cas2PersistedApplicationStatus(
         id = UUID.fromString("176bbda0-0766-4d77-8d56-18ed8f9a4ef2"),
         name = "placeOffered",
         label = "Place offered",
         description = "The applicant has been offered a place for Short-Term Accommodation (CAS-2).",
       ),
       Cas2PersistedApplicationStatus(
-        id = UUID.fromString("a919097d-b324-471c-9834-756f255e87ea"),
-        name = "onWaitingList",
-        label = "On waiting list",
-        description = "The applicant has been added to the waiting list for Short-Term Accommodation (CAS-2).",
+        id = UUID.fromString("fe254d88-ce1d-4cd8-8bd6-88de88f39019"),
+        name = "offerAccepted",
+        label = "Offer accepted",
+        description = "The accommodation offered has been accepted.",
       ),
       Cas2PersistedApplicationStatus(
```

Comment:
could not be placed and incomplete aren't mentioned in the ticket. presumably these are ok to be there?

Reply:
good spot thanks, but yep those ones are meant to be in there - they have `isActive` currently set to `false` so they won't be returned to UI.